blockbase/block-template-parts/header.html

```diff
@@ -1,5 +1,5 @@
-<!-- wp:group {"className":"site-header","layout":{"type":"flex"},"style":{"spacing":{"padding":{"bottom":"80px"}}}} -->
-<div class="wp-block-group site-header" style="padding-bottom:80px">
```

Comment:
The 'vertical' gap is defined as `"vertical": "min(30px, 5vw)"` (the dynamic-ness being something I'm sad to lose).  

But that means the top gap should probably default to 30px rahter than 40px as defined here.